portal-api/src/v1/users/routes.js

```diff
@@ -154,31 +155,38 @@ module.exports.findById = (req, res, next) => {
 };
 
 module.exports.updateById = (req, res, next) => {
-  findOne(req.params._id, (error, user) => {
+  const userIsAdmin = req.user.roles.includes(ADMIN);
```

Comment:
Can we add optional chaining here (req.user?.roles?.includes(ADMIN)) and the line below? This will stop exceptions being thrown is any of these are undefined in the request.

Likewise is it worth wrapping this in a try catch block and returning a 500 if any exceptions are thrown? (just makes it easier to debug if we catch errors, log them out then return a 500 to the user so that they have a sensible status code sent back to them)